### PR TITLE
[1LP][RFR] Adding BZ coverage

### PR DESCRIPTION
**Pull request by dgaikwad**
No test run required just adding BZ coverage

### CODE REVIEW

File: cfme/tests/automate/test_automate_manual.py [hunk 1]
```diff
@@ -376,3 +376,19 @@ def test_copy_automate_method_without_edit():
             3. You should be able to copy the highlighted text
     """
     pass
+
+
+@pytest.mark.tier(1)
```
Comment (john-dupuy) [inline cfme/tests/automate/test_automate_manual.py:381]:
```suggestion
@pytest.mark.tier(1)
@pytest.mark.meta(coverage=[1789706])
```

File: cfme/tests/automate/test_simulation.py [hunk 2]
```diff
@@ -195,6 +195,7 @@ def test_attribute_value_message(custom_instance):
     """
     Bugzilla:
         1753523
+        1740761
```
Comment (john-dupuy) [inline cfme/tests/automate/test_simulation.py:198]: Is this providing test coverage for 1740761?

Comment (dgaikwad) [reply cfme/tests/automate/test_simulation.py:198]: yes

File: cfme/tests/configure/test_access_control.py [hunk 3]
```diff
@@ -2835,3 +2837,52 @@ def test_tenant_template_visibility():
...
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testSteps:
+            1.Set a new default tenant name, e.g. 'Bit63'
+            2.From Rails console, lookup the corresponding tenant object, i.e.
+        expectedResults:
+            1.
+            2.tenant name should be updated on rail console
```
Comment (john-dupuy) [inline cfme/tests/configure/test_access_control.py:2886]:
```suggestion
            1. Set a new default tenant name, e.g. 'Bit63'
            2. From Rails console, lookup the corresponding tenant object, i.e.
        expectedResults:
            1.
            2. tenant name should be updated on rail console
```
Some additional about the commands used in the rails console would be helpful here.

File: cfme/tests/configure/test_access_control.py [hunk 4]
```diff
@@ -2835,3 +2837,52 @@ def test_tenant_template_visibility():
...
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testSteps:
+            1.Navigate to compute -> Cloud -> Tenants
+            2.Go to Configuration -> Create cloud tenant
+        expectedResults:
+            1.
+            2.It should show only one provider name.
```
Comment (john-dupuy) [inline cfme/tests/configure/test_access_control.py:2861]:
```suggestion
            1. Navigate to compute -> Cloud -> Tenants
            2. Go to Configuration -> Create cloud tenant
        expectedResults:
            1.
            2. It should show only one provider name.
```

File: cfme/tests/cli/test_appliance_console.py [hunk 5]
```diff
@@ -1796,3 +1796,55 @@ def test_appliance_console_logfile_config_reboot():
...
+            3.select "configure database"
+            4.select "Join Region in External Database"
+            5.enter invalid database name
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.Error information gets logged to appliance_console.log
+    """
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py:1850]:
```suggestion
    """
    pass
```

File: cfme/tests/cli/test_appliance_console.py [hunk 6]
```diff
@@ -1796,3 +1796,55 @@ def test_appliance_console_logfile_config_reboot():
...
+            2.press any key
+            3.select "configure database"
+            4.select "Join Region in External Database"
+            5.enter invalid database name
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.Error information gets logged to appliance_console.log
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py:1849]:
```suggestion
        testSteps:
            1. Execute "ap" command on shell
            2. press any key
            3. select "configure database"
            4. select "Join Region in External Database"
            5. enter invalid database name
        expectedResults:
            1.
            2.
            3.
            4.
            5. Error information gets logged to appliance_console.log
```

File: cfme/tests/cli/test_appliance_console.py [hunk 7]
```diff
@@ -1796,3 +1796,55 @@ def test_appliance_console_logfile_config_reboot():
...
+        initialEstimate: 1/3h
+        testSteps:
+            1. execute command on shell "ap"
+            2. press any key
+            3. check console
+        expectedResults:
+            1.
+            2.
+            3.  "translation missing: en.advanced_settings.timezone" should not be displayed
+    """
```
Comment (john-dupuy) [inline cfme/tests/cli/test_appliance_console.py:1822]:
```suggestion
    """
    pass
```

File: cfme/tests/integration/test_auth_manual.py [hunk 8]
```diff
@@ -607,3 +607,114 @@ def test_openid_auth_provider():
...
+            then users in that group will be able to see the WAS Key Pairs
+            5. Otherwise, if the added groups belong to one of the sub-tenant (Children tenant),
+             users in those groups will not be able to see the AWS Key Pairs
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.Users in groups that belong to either the top-level tenant or any of the
+            children tenants (sub-tenants) should be able to see the key Pairs.
```
Comment (john-dupuy) [inline cfme/tests/integration/test_auth_manual.py:718]:
```suggestion
            5. Users in groups that belong to either the top-level tenant or any of the
            children tenants (sub-tenants) should be able to see the key Pairs.
```

File: cfme/tests/integration/test_auth_manual.py [hunk 9]
```diff
@@ -607,3 +607,114 @@ def test_openid_auth_provider():
...
+    also check error in production.log log file
+    Bugzilla:
+        1803952
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: critical
+        initialEstimate: 1/4h
+        testSteps:
+            1. Go to configures--> Acesss controls
```
Comment (john-dupuy) [inline cfme/tests/integration/test_auth_manual.py:681]:
```suggestion
            1. Go to Configuration--> Access Control
```

File: cfme/tests/integration/test_auth_manual.py [hunk 10]
```diff
@@ -607,3 +607,114 @@ def test_openid_auth_provider():
...
+        caseposneg: negative
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testSteps:
+            1.Navigate to self service UI login page
+            2.Click login without entering any creds
+        expectedResults:
+            1.
+            2.Clicking the login button without entering credentials should have the
+            error flash message appear immediately
```
Comment (john-dupuy) [inline cfme/tests/integration/test_auth_manual.py:632]:
```suggestion
        testSteps:
            1. Navigate to self service UI login page
            2. Click login without entering any creds
        expectedResults:
            1.
            2. Clicking the login button without entering credentials should have the
            error flash message appear immediately
```

File: cfme/tests/automate/test_automate_manual.py [hunk 11]
```diff
@@ -376,3 +376,34 @@ def test_copy_automate_method_without_edit():
...
+@pytest.mark.meta(coverage=[1789806])
+def test_retire_vm_error_on_new_user():
+    """
+    There should be any log error in evm.log file while retiring the vm by new user
+    Bugzilla:
+        1789806
+    Polarion:
```
Comment (digitronik) [inline cfme/tests/automate/test_automate_manual.py:388]: looks like negative test
can we add `caseposneg` field?

File: cfme/tests/cli/test_appliance_console.py [hunk 12]
```diff
@@ -1796,3 +1796,57 @@ def test_appliance_console_logfile_config_reboot():
...
+        caseimportance: critical
+        initialEstimate: 1/3h
+        testSteps:
+            1. execute command on shell "ap"
+            2. press any key
+            3. check console
+        expectedResults:
+            1.
+            2.
+            3.  "translation missing: en.advanced_settings.timezone" should not be displayed
```
Comment (digitronik) [inline cfme/tests/cli/test_appliance_console.py:1821]: white space ^^

File: cfme/tests/cli/test_appliance_console.py [hunk 13]
```diff
@@ -1796,3 +1796,57 @@ def test_appliance_console_logfile_config_reboot():
...
+
+@pytest.mark.manual
+@pytest.mark.tier(0)
+@pytest.mark.meta(coverage=[1625377])
+def test_appliance_console_db_fail_log():
+    """
+    Error information gets logged to appliance_console.log on db configuration failure
+    Bugzilla:
+        1625377
+    Polarion:
```
Comment (digitronik) [inline cfme/tests/cli/test_appliance_console.py:1834]: this also looks like negative test

**Pull request merged**